.github/workflows/tests.yml

```diff
@@ -66,7 +66,7 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        python -m pip install --upgrade tox tox-gh-actions testfixtures
+        python -m pip install --upgrade 'tox>=3,<4' tox-gh-actions testfixtures
```

Comment:
Had to do that to make the tests pass, tox v4 was released recently and it's not working